### PR TITLE
Mepts | 1006 MQ New Template 2022

### PR DESCRIPTION
**Pull request by eliazmuchave**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/QualityImprovement2020DataSet.java [hunk 1]
```diff
@@ -2044,105 +2045,105 @@ public DataSetDefinition constructQualityImprovement2020DataSet() {
         "age=10-14");
 
     // Category 15 Denominator MDS
-    CohortIndicator MQ15DenMds =
+    CohortIndicator MQ15DEN13  =
         eptsGeneralIndicator.getIndicator(
-            "MQ15DenMds",
+            "MQ15NUM13",
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/QualityImprovement2020DataSet.java]:
```suggestion
            "MQ15DEN13",
```

Comment (psmatsinhe) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/QualityImprovement2020DataSet.java]: I think here you meant DEN not NUM

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupQualityImprovementReport20.java [hunk 2]
```diff
@@ -76,7 +77,7 @@ public ReportDefinition constructReportDefinition() {
       reportDesign =
           createXlsReportDesign(
               reportDefinition,
-              "Melhoria de Qualidade - 2020.xls",
+              "Template_MQ_2022.xls",
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupQualityImprovementReport20.java:80]: This it the name that will show-up in the generated report, I think is much better to name it Melhoria de Qualidade - 2022.xls

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
@eliazmuchave please have a look

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/QualityImprovement2020DataSet.java (hunk 1 above)
Comment (eliazmuchave) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/QualityImprovement2020DataSet.java]: I fixed it

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupQualityImprovementReport20.java (hunk 2 above)
Comment (eliazmuchave) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupQualityImprovementReport20.java:80]: It's a template name, the generated report name is "Melhoria de Qualidade - 2022"

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**